helm-chart/README.md

```diff
@@ -122,6 +122,8 @@ helm install kubeshark kubeshark/kubeshark \
 | `tap.release.name`                        | Helm release name                          | `kubeshark`                                             |
 | `tap.release.namespace`                   | Helm release namespace                | `default`                                               |
 | `tap.persistentStorage`                   | Use `persistentVolumeClaim` instead of `emptyDir` | `false`                                                |
+| `tap.persistentStorageStatic`          | Use static persistent volume provisioning (explicitly defined `PersistentVolume` ) | `false`                                                      |
```

Comment:
@tetatetit is the `tap.persistentStorageStatis` related to EFS or in general to PV? If this is generic, than its fine. IF it's EFS specific, the name should refer to EFS.

Reply:
This also needs to be addressed in the documentation.

Reply:
> This also needs to be addressed in the documentation.

Done https://github.com/kubeshark/docs/pull/18